ModelLibrary/Model/MPaymentTerm.cs

```diff
@@ -1171,6 +1171,13 @@ protected override bool BeforeSave(bool newRecord)
                 }
             }
 
+            //** If schedule line are present system should not allow to save header with week day ** Dt: 02/04/2021 ** Modified By: Kumar ** //
+            if (Util.GetValueOfInt(GetWeekOffset()) > 0 && count > 0)
+            {
+                log.SaveError("Error", Msg.GetMsg(GetCtx(), "VIS_ScheduleExist"));
```

Comment:
log.SaveError("VIS_ScheduleExist", null);

Reply:
As discussed, modified it as below.
log.SaveError("VIS_ScheduleExistForWeekDay", string.Empty);